docs/terminal/shell-integration.md

```diff
@@ -81,6 +81,16 @@ code --locate-shell-integration-path bash
 [[ "$TERM_PROGRAM" == "vscode" ]] && . "/path/to/shell/integration/script.sh"
 ```
 
+## Shell integration quality
+
+When using shell integration, it has a "quality" associated with it that declares the capabilities of it. These qualities are determined by how the shell integration script behaves.
+
+- **None**: No shell integration is active
+- **Rich**: Shell integration is active and command detection is working in an ideal way.
+- **Basic**: Shell integration is active but command detection may not be ideal. This typically means the shell has very basic command detection that knows where a command was run but not what it is or what the exit status was.
```

Comment:
```suggestion
- **Basic**: Shell integration is active, but command detection might be limited: the command run location is detected, but not its exit status.
```

Reply:
Suggested rewording - if you prefer to keep yours, do replace "may" with "might".

Reply:
I read that as this is exactly the case when it's basic but that's not true, basic is a spectrum from very basic up until rich.